api/v1alpha1/openapi.yaml

```diff
@@ -3756,8 +3756,26 @@ components:
           description: A list of application content.
           items:
             $ref: "#/components/schemas/ApplicationContent"
+        volumes:
```

Comment:
Since this PR adds a new field to an existing object, we'd need an adaptation in the UI to make sure that for inline applications that have defined volumes, they won't get removed as the field wouldn't be included in the PATCH request the UI sends to update the application.

Also, we may need to discuss whether this is only a API/CLI feature, or it should also be included in the UI.

I imagine this feature is not for 0.7?

Reply:
correct this is a sketch for some work @rawagner has been working on nd not planned for 0.7